src/v3/src/components/Widget/style.css

```diff
@@ -179,3 +179,8 @@ span.strong {
 span.no-translate {
   white-space: nowrap;
 }
+
+.app-store-link svg {
+  width: fit-content !important;
+  height: fit-content !important;
```

Comment:
These values should be passed into the width and height properties of the interface for the ImageLink component, instead of as a class. 

The reason for this is because of the deprecation of the use of classes within the SIW and to allow customers to easily customize the widget via the Odyssey/MUI theme/token overrides.

Reply:
without these values in the css class the svg is forced to 1em/1em size:
<img width="1190" alt="Screenshot 2024-02-29 at 18 50 30" src="https://github.com/okta/okta-signin-widget/assets/1379394/1ddbdeb8-8a9a-447f-835b-7781226bd02e">

Reply:
please see https://github.com/okta/okta-signin-widget/pull/3526/commits/081366e9ba882712767a9b67c1ae4b299a7cdbfa